content/blog/2023-07-07-newsletter.md

```diff
@@ -0,0 +1,181 @@
+### Clients and applications
+
+- [JMP is launched and out of Beta](https://blog.jmp.chat/b/launch-2023)
+- [Cheogram Android version `2.12.1-6`](https://git.singpolyma.net/cheogram-android/refs/2.12.1-6) adds per account colours, per account quiet hours, thumbhash previews of un-downloaded files, and more.
+- [Libervia progress note 2023-W22](https://www.goffi.org/b/libervia-progress-note-2023-w22-x3Wa) on A/V Implementation, ActivityPub Gateway and more.
+- [Dino 0.4.3](https://github.com/dino/dino/releases/tag/v0.4.3).
+- [Movim 0.22.2 is out](https://mov.im/node/pubsub.movim.eu/Movim/1d407d21-989d-404b-9a39-aa9b328882d1).
+
+### Servers
+
+- The ejabberd [mod_s3_upload](https://github.com/processone/ejabberd-contrib/tree/master/mod_s3_upload) module gained [support for the use of a separate download host](https://github.com/processone/ejabberd-contrib/pull/320). This allows clients to download media content from a statically hosted S3 bucket. This was initially proposed to allow ejabberd to integrate 
+with [Garage](https://garagehq.deuxfleurs.fr/), an open-source distributed object storage service tailored for self-hosting.
+
+### Libraries & Tools
+
+- New website and [xmpp.rs version 0.4 release](https://xmpp.rs/blog/release-xmpp-0-4/): After more than 3 years and half since the last release (version 0.3 in septembre 2019), xmpp-rs developers are happy to announce xmpp-rs version 0.4.0. If you don't know about xmpp-rs, those are Rust crates for interoperable and federated instant messaging and other social networking applications.
+- [omemo_dart](https://github.com/PapaTutuWawa/omemo_dart) released [version 0.5.0](https://github.com/PapaTutuWawa/omemo_dart/releases/tag/v0.5.0), which finally allowed Moxxy to establish stable OMEMO:2 sessions between two Moxxy clients.
+- [Openfire inVerse plugin v10.1.4-1 release!](https://discourse.igniterealtime.org/t/openfire-inverse-plugin-v10-1-4-1-release/92969)
+- From the ignite realtime community:
+  - [Candy plugin for Openfire 2.2.0 Release 4 now available](https://discourse.igniterealtime.org/t/candy-plugin-for-openfire-2-2-0-release-4-now-available/93042)
+  - [Search Openfire plugin 0.7.4 release](https://discourse.igniterealtime.org/t/search-openfire-plugin-0-7-4-release/93043)
+  - [Push Notification Openfire plugin 0.9.2 released](https://discourse.igniterealtime.org/t/push-notification-openfire-plugin-0-9-2-released/93044)
+  - [JmxWeb plugin for Openfire 0.9.1 release](https://discourse.igniterealtime.org/t/jmxweb-plugin-for-openfire-0-9-1-release/93045)
+  - [Agent Information plugin for Openfire release 1.0.1](https://discourse.igniterealtime.org/t/agent-information-plugin-for-openfire-release-1-0-1/93057/1)
+  - [Jabber Browsing Openfire Plugin 1.0.1 released](https://discourse.igniterealtime.org/t/jabber-browsing-openfire-plugin-1-0-1-released/93059)
+- [A feature-rich Matrix to XMPP puppeteering gateway, based on slidge and nio.](https://git.sr.ht/~nicoco/matridge)
```

Comment:
This entry should be moved above the OpenFire entries (otherwise it gets lost)

Reply:
7f18714ba71e36d73936ace93ca8bbce1e91b5eb

Thanks for reviewing!